### PR TITLE
Update Microsoft.Azure.WebJobs.Host.Storage version

### PR DESCRIPTION
**Pull request by liliankasem**
The change to fix the lease log issue was made to `Microsoft.Azure.WebJobs.Host.Storage` package, bumping the version to get that [change](https://github.com/Azure/azure-webjobs-sdk/pull/2819) in.

### CODE REVIEW

File: src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj [hunk 1]
```diff
@@ -74,8 +74,8 @@
     <PackageReference Include="Microsoft.Azure.AppService.Middleware.Functions" Version="1.4.17" />
     <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="1.0.8" />
     <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.1.7" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.34-11937" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="5.0.0-beta.1" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.35-11939" />
```
Comment (mathewc) [inline src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj]: Not sure we should be at minor version 35. In [this](https://github.com/Azure/azure-webjobs-sdk/commit/71c2262d8bf64fb845429c557732f9be0e7b24a8) PR I recently bumped us to 34 and we haven't officially released that yet, so we should still be in the 34 minor version train.

Comment (liliankasem) [reply src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj]: Reverting to 34!

File: src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj [hunk 2]
```diff
@@ -75,7 +75,7 @@
     <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="1.0.8" />
     <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.1.7" />
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.34-11937" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="5.0.0-beta.1" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="5.0.0-beta.2" />
```
Comment (mathewc) [inline src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj]: Where does this version exist? I only see [5.0.0-beta.2-11939](https://www.myget.org/feed/azure-appservice/package/nuget/Microsoft.Azure.WebJobs.Host.Storage/5.0.0-beta.2-11939) on the feed? Shouldn't you be referencing that version?

Comment (liliankasem) [reply src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj]: Changed to `5.0.0-beta.2-11939` - was going to hold off this PR till we got the official nuget release for 5.0.0-beta.2 but as discussed offline, we should be good to just use the prerelease!

**Review by mathewc — APPROVED** [review]

**Review by satvu — APPROVED** [review]

**Pull request merged**